package.json

```diff
@@ -11,9 +11,9 @@
     "prettier": "1.10.2"
   },
   "lint-staged": {
-    "*.{js,jsx}": ["eslint --ignore-path .prettierignore --fix", "git add"],
-    "*.json": ["prettier --write", "git add"],
-    "*.md": ["prettier --write", "git add"],
+    "*.{js,jsx}": ["eslint --ignore-path .prettierignore-jsScript --fix", "git add"],
+    "*.json": ["prettier --ignore-path .prettierignore-config --write", "git add"],
+    "*.md": ["prettier --ignore-path .prettierignore-docs --write", "git add"],
```

Comment:
I don't get it.. why do we have three different ignore files?

Reply:
Because Prettier doesn't support having multiple ignore patterns.

Reply:
w00t? but why not having one ignore file and put all the rules there..?

Reply:
Let's try